### ISSUE TITLE
TableIndex does not seem to be creating indexes when used in different file

### ISSUE THREAD

**Issue by 23doors** [simolus3/moor#2645]
**Describe the bug**

Example model:
```
@TableIndex(name: "cache_images_added_at_persist", columns: {#addedAt})
class CacheImages extends Table {
  IntColumn get id => integer().autoIncrement()();
  TextColumn get key => text().unique()();
  DateTimeColumn get addedAt => dateTime()();
  BoolColumn get persist => boolean()();
}
```

build runner succeeds, but seems that index is not created after I recreate the database by deleting whole database file.

Checked it by opening sqlite3 db manually and listing all indexes:
```
SELECT name 
FROM sqlite_master 
WHERE type = 'index';
```

Also checked log_statements=true during startup and no CREATE INDEX is listed.

**Comment by 23doors** [comment]
Checked a bit more.
If I have @DriftDatabase in same file as table definitions and @TableIndex - then it works fine.

But if I have tables defined in separate .dart file and imported - then create index isn't getting created and is missing from allSchemaEntities

**Comment by simolus3** [comment]
Thanks for the report! This should be fixed in 4eb44f9623c94ddc90cc287975e29f935dd4e35c.

As a workaround, you could try adding the Dart file with the index to the `includes` section of the `@DriftDatabase` annotation, e.g. `@DriftDatabase(tables:  [CacheImages], include: {'path/to/table.dart'})`.
